lib/takeaway.rb

```diff
@@ -0,0 +1,69 @@
+require_relative 'confirmation'
+
+class Takeaway
+attr_reader :menu, :selected_dishes
+
+  def initialize
+    @menu = {"VEGAN BURGER"=> 10.95, "VEGAN CHILLI" => 11.95, "CHIPS" => 4.50}
+    @selected_dishes = []
+    @total = 0
+  end
```

Comment:
Your Takeaway class is in charge of:
- knowing the menu
- knowing the ordered items
- manage total

To follow DRY you could separate the above responsibilities